Add Thunder Client to recommended extensions (#288)
UMM-CSci-3601/3601-lab2_client-server: LABTASKS.md

- [Notes on notation and structure](#notes-on-notation-and-structure)
- [Exploring the project](#exploring-the-project)
- [Exploring the server](#exploring-the-server)
  - [Using Thunder Client](#using-thunder-client)

>     - [Usage](#usage)

Reviewer: Make sure to update the ToC (i.e., remove the "Usage" entry).

If you're using the Markdown All In One VSCode extension, it should manage that automatically, but if you make changes here in GitHub then you've got to manage that stuff as well.

to provide coverage of the `Server` class, but we haven't tried that
yet. (You can, though. 😄)

> ### Using Thunder Client

Reviewer 2: ```suggestion
### Use Thunder Client to explore API output
```

> Thunder client is a tool for debugging the server API output from VSCode. 

Reviewer 2: ```suggestion
Thunder Client is a tool for debugging the server API output from VSCode. 
```

Reviewer: I agree that we want to be more consistent in the way we write this (something I'm never good at, TBH).

It aids in checking what the server gives us when we make requests to it, which can be 
really helpful when you're trying to debug what your server gives you.

> #### Usage

Reviewer: I don't think it makes sense to have this `Usage` sub-header, especially when when it's in a section that is about _using_ Thunder Client that has almost no other content.

```suggestion
```

> To use Thunderclient (once it's installed), open it from the sidebar. 

Reviewer 2: ```suggestion
To use Thunder Client (once it's installed), open it from the sidebar. 
```

The icon is a circle with a lightning bolt in the middle.

> This should add a button in the top of the sidebar labelled `New Request`, click it.  

Reviewer: I'm not sure why, but this line seems to quasi-end a paragraph, but not quite? The next line starts on a new line (instead of extending this one), but doesn't seem to start a new paragraph? Did you embed a carriage return or something like that by accident?

Reviewer: This would also be a good place to add screenshots. @helloworld12321 and @floogulinc have been working to add more screenshots to the write-ups, and I do think they help quite a lot. There's a risk of them aging badly, but that hasn't seemed to have been a huge issue, and I think they convey a lot of useful info to the students.
